core/src/main/resources/error/error-classes.json

```diff
@@ -191,6 +196,11 @@
     ],
     "sqlState" : "0AKD0"
   },
+  "CANNOT_RESOLVE_STAR_EXPAND" : {
+    "message" : [
+      "Cannot resolve '<targetString>.*' given input columns '<columns>'. Please check that the specified table or struct exists and is accessible in the input columns."
```

Comment:
Incorrect quoting of <columns>. Please, remove '' and use `toSQLId` (if they are ids).